### ISSUE TITLE
blink.py error: unexpected keyword argument 'max'

### ISSUE THREAD

**Issue by pdp7** [icebreaker-fpga/icebreaker-amaranth-examples#3]
I encountered this error:
```
pdp7@x1:~/dev/fpga/icebreaker-nmigen-examples/blink$ python3 ./blink.py 
Traceback (most recent call last):
  File "./blink.py", line 34, in <module>
    plat.build(Blinker(10000000), do_program=True)
  File "/home/pdp7/.local/lib/python3.7/site-packages/nmigen/build/plat.py", line 78, in build
    plan = self.prepare(elaboratable, name, **kwargs)
  File "/home/pdp7/.local/lib/python3.7/site-packages/nmigen/build/plat.py", line 118, in prepare
    fragment = Fragment.get(elaboratable, self)
  File "/home/pdp7/.local/lib/python3.7/site-packages/nmigen/hdl/ir.py", line 39, in get
    obj = obj.elaborate(platform)
  File "./blink.py", line 16, in elaborate
    counter = Signal(max=self.maxperiod+1)
TypeError: __init__() got an unexpected keyword argument 'max'
```

**Comment by pdp7** [comment]
I tried this:
```diff
diff --git a/blink/blink.py b/blink/blink.py
index 8e66c9f..0a828f4 100644
--- a/blink/blink.py
+++ b/blink/blink.py
@@ -13,8 +13,8 @@ class Blinker(Elaboratable):
         m.domains.sync = ClockDomain()
         m.d.comb += ClockSignal().eq(clk12)
         
-        counter = Signal(max=self.maxperiod+1)
-        period = Signal(max=self.maxperiod+1)
+        counter = Signal(self.maxperiod+1)
+        period = Signal(self.maxperiod+1)
         
         m.d.comb += period.eq(self.maxperiod)         
```
It does run but no blinking LED like with the migen example.

**Comment by pdp7** [comment]
note: I never see the bitstream programming output

**Comment by pdp7** [comment]
for contrast, this works:
```
pdp7@x1:~/dev/fpga/icebreaker-nmigen-examples/7seg_count$ python3 ./7seg_count.py 
init..
cdone: high
reset..
cdone: low
flash ID: 0xEF 0x40 0x18 0x00
file size: 104090
erase 64kB sector at 0x000000..
erase 64kB sector at 0x010000..
programming..
reading..
VERIFY OK
cdone: high
Bye.
```

**Comment by pdp7** [comment]
solved by @miek in #4
